specification/http/1.0/openapi.json

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
Confirming that convention is snake_casing for all types?

Reply:
That's what we have standardized on internally; but I am impartial myself. What do you think?

Reply:
That works! My original question was incorrect, the convention is:

* attributes as snake_cased
* Types are CamelCased